dash.el

```diff
@@ -1170,20 +1170,28 @@ returns the header value, but only after seeing at least one
 other value (the body)."
   (--partition-by-header (funcall fn it) list))
 
-(defun -partition-after-pred (pred list)
+(defmacro -partition-after-pred (pred list)
   "Partition directly after each time PRED is true on an element of LIST."
-  (when list
-    (let ((rest (-partition-after-pred pred
-                                       (cdr list))))
-      (if (funcall pred (car list))
-          ;;split after (car list)
-          (cons (list (car list))
-                rest)
-
-        ;;don't split after (car list)
-        (cons (cons (car list)
-                    (car rest))
-              (cdr rest))))))
+  (declare (debug (pred pred)))
```

Comment:
AFAIK `pred` isn't a valid Edebug spec; see [`(info "(elisp) Instrumenting Macro Calls")`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Instrumenting-Macro-Calls.html) for info on `debug` declarations.

Reply:
Good catch.

Reply:
I removed it now, so should be OK.